File: steven/pert12/backend9/models/event.go
package models

import "time"

type Event struct {
	ID        int       `json:"id"    gorm:"primaryKey"`
	Location  string    `json:"location"  gorm:"type:varchar(100)"`
	Start     time.Time `json:"start" gorm:"type:datetime;default:CURRENT_TIMESTAMP"`
	Cover     string    `json:"cover" gorm:"type:varchar(100);default('https://i.pravatar.cc/150')"`
}

func (*Event) TableName() string {
	return "events"
}

File: m-naufal-rafif-pratama/tugas12/frontend/src/types/event.ts
export type Event = {
  id: number;
  location: string;
  start: string;
  cover: string;
};

export type EventResponse = {
  events: Event[]
};
export type EventCardProps = {
  id: number;
  title: string;
  date: string;
  description: string;
  image: string;
};